### PR TITLE
Makes xeno spit inject a fast acting neurotoxin instead of instantly stunning

### PR DESCRIPTION
**Pull request by Moltijoe**
in my opinion, one PR overnerfs it and the other doesn't do enough

This takes the idea of it having a delayed effect, but tweaks it to be more deterministic and leaves it as ranged

It also scales with permeability coefficient, so once that gets figured out by someone else, i'll let people properly wear clothing to protect themselves

they can also take charcoal to try and purge it before it does too much damage

credited burger because i basically just copy pasted their neurotoxin and then tweaked it a bit

alternatives to these
closes: #18537
closes: #18549

:cl:  @BurgerLUA @Moltijoe 
tweak: Xeno spit injects a fast acting paralysis chemical instead of stunning
/:cl:

### CODE REVIEW

**Comment by 00ze-cyclone** [comment]
Why make it a drink subtype when it’ll never end up in a glass anyway? Why not just a toxin

**Comment by Moltijoe** [comment]
> Why make it a drink subtype when it’ll never end up in a glass anyway? Why not just a toxin

no clue, i just copied it from burger's PR

**Comment by Marmio64** [comment]
queen, prae, and sentinel should get some kind of buff in return for being substantially weaker from any neurotoxin nerf. The spit is a lifesaver ability that prevents them from just getting dominated from range so they'd really need higher survivability to try to last long enough for the negative effects to set in. A 50 health buff to praetorian and queen, and a 25 health buff to sentinel would probably help a lot.

**Comment by Moltijoe** [comment]
> queen, prae, and sentinel should get some kind of buff in return for being substantially weaker from any neurotoxin nerf. The spit is a lifesaver ability that prevents them from just getting dominated from range so they'd really need higher survivability to try to last long enough for the negative effects to set in. A 50 health buff to praetorian and queen, and a 25 health buff to sentinel would probably help a lot.

I wouldnt be opposed to buffing health, though id make that a different PR

File: yogstation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -74,3 +74,43 @@
...
+	boozepwr = 25
+	quality = DRINK_VERYGOOD
+	taste_description = "a numbing sensation"
+	metabolization_rate = 1 * REAGENTS_METABOLISM
+	glass_icon_state = "neurotoxinglass"
+	glass_name = "Neurotoxin"
+	glass_desc = "A drink that is guaranteed to knock you silly."
+	var/list/bodyparts = list(TRAIT_PARALYSIS_L_ARM, TRAIT_PARALYSIS_R_ARM, TRAIT_PARALYSIS_R_LEG, TRAIT_PARALYSIS_L_LEG)
+
+/datum/reagent/consumable/ethanol/neurotoxin_alien/proc/pickt()
```
Comment (ThatLing) [inline yogstation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Use proper proc and var names

**Review by ThatLing — DISMISSED** [review]

**Comment by Moltijoe** [comment]
> Why make it a drink subtype when it’ll never end up in a glass anyway? Why not just a toxin

i've done just that now

**Comment by Therandomhoboo** [comment]
Wait.... we nerfing the drink neurotoxin?

**Comment by Moltijoe** [comment]
> Wait.... we nerfing the drink neurotoxin?

no?

**Review by ToasterBiome — APPROVED** [review]

**Comment by ToasterBiome** [comment]
lets see how it goes

**Pull request merged**